httomo/cli.py

```diff
@@ -59,11 +60,11 @@ def run(
     reslice_dir: Union[Path, None],
 ):
     """Run a pipeline defined in YAML on input data."""
-    
-    # First we need to validate yaml configuration file if there are any errors
-    # TODO: with new yaml syntax check yaml is not fully working.
-    # Need to re-enable that: 
-    # _check_yaml(yaml_config, in_data_file)
+    if yaml_config.suffix == ".yaml":
+        try:
+            assert _check_yaml(yaml_config, in_data_file) == True
```

Comment:
A nitpick, feel free to take it or leave it. I think the checking of equality of the returned boolean value of `_check_yaml()` is unnecessary here? 

Ie, I think this is enough:
```python
assert _check_yaml(yaml_config, in_data_file)
```

Reply:
Thank you for highlighting this, updated as suggested in [822a202](https://github.com/DiamondLightSource/httomo/pull/201/commits/822a2023491ab8d0b01d76577c15590e6a79b0ae). However, then this piece of code was removed following discussion on at which point yaml checker will be run.